regression_tests/pull_request_tests.sh

```diff
@@ -28,19 +28,26 @@ aws configure --profile "$AWS_PROFILE" set account_id "$AWS_ACCOUNT_ID"
 aws configure --profile "$AWS_PROFILE" set region "eu-west-2"
 aws configure --profile "$AWS_PROFILE" set output "json"
 
-echo -e "\nRun platform-helper copilot make-addons"
-PLATFORM_TOOLS_SKIP_VERSION_CHECK=true platform-helper copilot make-addons
+echo -e "\nRun platform-helper generate"
+PLATFORM_TOOLS_SKIP_VERSION_CHECK=true platform-helper generate
```

Comment:
We are now guarding slightly more stuff against regressions.

Reply:
It does, and is delegate to by `platform-helper generate`. I have tweaked the verbiage to make it clearer that it's being exercised here.